### ISSUE TITLE
Kontaktní adresa v registračním formuláři

### ISSUE THREAD

**Issue by zlopocasna** [LIBCAS/BankID-Registrator#21]
webový i mobilní
kontaktní adresa je po načtení formuláře zaškrtnutá a zobrazená; v základním nastavení má být skrytá

mobilní
Při mazání předvyplněné adresy se po smazání obsahu pole ulice objeví našeptávač adres; ten překryje další pole, které v tu chvíli není možné smazat. Pokud tam chce žadatel něco vyplňovat, tak je to fajn, ale v případě, že chce adresu jen vymazat, je to vopruz

**Comment by TOA-Anakin** [comment]
@zlopocasna 

- Kontakní adresu jsem již dal ať je nezaškrtnutá by default
- Pokud mažeš políčko s ulicí, tak to program bere jako změnu adresy a proto vyskočí ten našeptávač, stačí klinout kamkoliv jinam a našeptávač zmizí. Navrhuji přidat pro každé pole tlačítko pro vymazání jeho obsahu pokud pole není prázdné aby nemusel mazat znak po znaku, souhlasíš?

**Comment by zlopocasna** [comment]
@TOA-Anakin 
Jo, tu funkčnost jsme pochopili, ale shodli jsme se na tom, že je to nemotorné. Tvůj návrh je skvělé řešení. Ano, souhlasím.
Dík

**Comment by TOA-Anakin** [comment]
@zlopocasna Je to tam v obou formularich
